cmd/sops/subcommand/filestatus/filestatus.go

```diff
@@ -0,0 +1,47 @@
+package filestatus
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"go.mozilla.org/sops"
+	"go.mozilla.org/sops/cmd/sops/codes"
+	"go.mozilla.org/sops/cmd/sops/common"
+)
+
+// Opts rapresent input option for FileStatus
+type Opts struct {
+	InputStore sops.Store
+	// OutputStore       sops.Store
+	InputPath string
+	// KeyServices       []keyservice.KeyServiceClient
+	// KeyGroups         []sops.KeyGroup
+	// GroupThreshold    int
```

Comment:
Remove the commented lines since they're not used

Reply:
Removed in commit https://github.com/mozilla/sops/pull/545/commits/80686aba224a6fed3aec410d0f75fc50065d2384